engine.py

```diff
@@ -22,6 +22,40 @@ class MariEngine(sgtk.platform.Engine):
     """
     The engine class
     """
+
+    @property
+    def host_info(self):
+        """
+        :returns: A {"name": application name, "version": application version, "stage": application build info}
```

Comment:
Suggest changing to this:

```python
@property
def host_info(self):
    """
    :returns: A dictionary with information about the application hosting this engine. 

    The returned dictionary is of the form::
    
        {
            "name": "Mari",
            "version": "v1.2.3",
            "stage": "build info"
        }
    """
```

I wouldn't inlclude the mac-specific doc link. Also, do we need the `stage`? Maybe leave that out until we know we need it.

Reply:
Ok, I couldn't find a web-based version of the documentation